pkgs/development/python-modules/json-rpc/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonApplication
+, fetchPypi
+, tox
+}:
+buildPythonApplication rec {
```

Comment:
`buildPythonPackage` should be used for non-apps.

Reply:
My bad. Fixed